R/write.IAMCxlsx.R

```diff
@@ -22,19 +22,13 @@
 
 #' @export
 write.IAMCxlsx <- function(x, path, append = FALSE) {
-
+    x <- as.quitte(x)
+    if (nrow(x) == 0) warning("Empty data frame written to ", path)
```

Comment:
Interesting choice to report success at the head of a function rather then the bottom …